nvflare/fuel/flare_api/flare_api.py

```diff
@@ -275,6 +280,18 @@ def list_jobs(self, detailed: bool = False, all: bool = False) -> List[dict]:
             command = command + " -d"
         if all:
             command = command + " -a"
+        if limit:
+            command = command + " -m " + str(limit)
+        if name_prefix:
+            if not isinstance(name_prefix, str):
+                raise InvalidArgumentError("name_prefix must be str but got {}.".format(type(name_prefix)))
+            else:
+                command = command + " -n " + name_prefix
+        if id_prefix:
+            if not isinstance(id_prefix, str):
+                raise InvalidArgumentError("id_prefix must be str but got {}.".format(type(id_prefix)))
+            else:
+                command = command + " " + id_prefix
```

Comment:
To be consistent should we also check other arguments passed in to make sure they have the correct type?

Reply:
I can add checks for other args

Reply:
May do this in future PR though, going through more systematically. For now just getting this PR merged.